src/Integration.Vsix/SonarLintDaemon/Protocol.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.SonarLintDaemon
+{
+
+    class Request
```

Comment:
We're always explicit about the class scope. Here, `internal` would do i.e. only visible within this project, and to any projects we explicitly permit to see our internals.